typescript/sdk/src/token/EvmERC20WarpRouteReader.ts

```diff
@@ -662,6 +710,114 @@ export class EvmERC20WarpRouteReader extends EvmRouterReader {
     };
   }
 
+  private async deriveEverclearBaseBridgeConfig(
+    everclearTokenbridgeInstance: EverclearTokenBridge,
+  ): Promise<
+    Pick<
+      EverclearEthBridgeTokenConfig,
+      'everclearBridgeAddress' | 'outputAssets' | 'everclearFeeParams'
+    >
+  > {
+    const [everclearBridgeAddress, domains] = await Promise.all([
+      everclearTokenbridgeInstance.everclearAdapter(),
+      everclearTokenbridgeInstance.domains(),
+    ]);
+
+    const outputAssets = await promiseObjAll(
+      objMap(arrayToObject(domains.map(String)), async (domainId, _) =>
+        everclearTokenbridgeInstance.outputAssets(domainId),
+      ),
+    );
+
+    const feeParamsByDomain = await promiseObjAll(
+      objMap(arrayToObject(domains.map(String)), async (domainId, _) => {
+        const [fee, deadline, signature] =
+          await everclearTokenbridgeInstance.feeParams(domainId);
+
+        return {
+          deadline: deadline.toNumber(),
+          fee: fee.toNumber(),
+          signature,
+        };
+      }),
+    );
+
+    return {
+      everclearBridgeAddress,
+      // Remove unset domains from the output
```

Comment:
personally would not expect this to be in the return block but rather at the variable declaration